src/handlers/identity/resolve.rs

```diff
@@ -13,19 +13,15 @@ use {
     serde::{Deserialize, Serialize},
     serde_json::{json, Value},
     std::sync::Arc,
+    tracing::instrument,
     validator::Validate,
 };
 
-#[derive(Deserialize)]
+#[derive(Deserialize, Debug, Validate)]
 pub struct ResolveIdentityPayload {
     #[serde(rename = "publicKey")]
-    public_key: String,
-}
-
-#[derive(Validate, Debug)]
-pub struct ResolveIdentityParams {
     #[validate(custom = "validate_identity_key")]
-    identity_key: String,
+    public_key: String,
```

Comment:
why?

Reply:
You think that has a time impact?

Reply:
Cus don't need redundant structs

Reply:
No I don't think it impacts the time significantly